vfs/vfscache/item.go

```diff
@@ -811,6 +1009,10 @@ func (item *Item) _ensure(offset, size int64) (err error) {
 	}
 	r := ranges.Range{Pos: offset, Size: size}
 	present := item.info.Rs.Present(r)
+	/* This statement simulates a cache space error for test purpose */
```

Comment:
This could maybe be a private cache option that we can set during tests

Reply:
Sound a great idea! Could you help adding this cache option and the test triggers while I get the function implementation done to get it tested for the current beta?  I would like to do it myself, but it would probably take longer and affect the timeliness of the feature.

Reply:
OK carry on with the PR and at some point I'll merge into a branch we can both work on.

There is bound to be some stuff I'd like to add and that is quite a good way of working.

Reply:
> OK carry on with the PR and at some point I'll merge into a branch we can both work on.
> 
> There is bound to be some stuff I'd like to add and that is quite a good way of working.

I think this is a good idea!